templates/repo/settings/options.tmpl

```diff
@@ -217,6 +219,13 @@
 													<label for="push_mirror_password">{{.locale.Tr "password"}}</label>
 													<input id="push_mirror_password" name="push_mirror_password" type="password" value="{{.push_mirror_password}}" autocomplete="off">
 												</div>
+												<div class="inline field {{if .Err_PushMirrorUseSSH}}error{{end}}">
+													<div class="ui checkbox df ac-center">
```

Comment:
```suggestion
													<div class="ui checkbox df ac">
```

I think this is what you meant, `ac` does `align-content: center`, `ac-center` does not exist